docs/eventing/samples/writing-receive-adapter-source/05-receive-adapter.md

```diff
@@ -6,8 +6,7 @@ type: "docs"
 ---
 
 ## Receive Adapter cmd
-Similar to the controller, we'll need an injection based main.go similar to the controller
-under `cmd/receiver_adapter/main.go`
+Similar to the controller, we'll need an injection based `main.go` similar to the controller under `cmd/receiver_adapter/main.go`
 ```go
 // This Adapter generates events at a regular interval.
 package main
```

Comment:
Maybe using `knative.dev/eventing/pkg/adapter/v2` is suggested?

Reply:
That's a good point, but I would prefer leaving it for another PR explaining that `adapter/v2`.